analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -22,6 +22,8 @@
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
     <!-- Class WindowsFormsMetadataReference needs this FrameworkReference to generate metadata references for Windows Forms related test cases. -->
     <FrameworkReference Include="Microsoft.WindowsDesktop.App.WindowsForms" />
+    <!-- Class WpfMetadataReference needs this FrameworkReference to generate metadata references for WPF related test cases. -->
+    <FrameworkReference Include="Microsoft.WindowsDesktop.App.WPF" />
```

Comment:
This looks heavy for a single FP. It would be fine for WPF-related sprint and pack of rules.
Can you just scaffold empty classes in a namespace with he same name instead?

Reply:
I also wanted to add some framework types constructors/methods which are known to take namespace parameters like [XmlnsDictionary(System.Windows.Markup)](https://learn.microsoft.com/dotnet/api/system.windows.markup.xmlnsdictionary?view=windowsdesktop-6.0) which is in another assembly but also requires the WPF framework reference.

Do we have an MMF for WPF? If so I would add a comment to the definition of the attributes that they should be removed as part of the MMF.